hydromt/components/grid.py

```diff
@@ -155,8 +154,9 @@ def write(
     @hydromt_step
     def read(
         self,
-        fn: str = "grid/grid.nc",
+        filename: str = "grid/grid.nc",
```

Comment:
Is there still another PR that will look into these filenames? Otherwise I have to change it here to `Optional[str] = None`